Dockerfile

```diff
@@ -29,6 +29,17 @@ ONBUILD ARG PGUSER=odoo
 ONBUILD ARG PGPASSWORD=odoopassword
 ONBUILD ARG PGHOST=db
 ONBUILD ARG PGDATABASE=prod
+ONBUILD ARG ODOO_MAX_CRON_THREADS=2
+ONBUILD ARG ODOO_WORKERS=0
+ONBUILD ARG ODOO_LIMIT_TIME_REAL=120
+ONBUILD ARG ODOO_LIMIT_TIME_CPU=60
+ONBUILD ARG ODOO_LIMIT_MEMORY_HARD=805306368
+ONBUILD ARG ODOO_LIMIT_MEMORY_SOFT=671088640
+ONBUILD ARG ODOO_LIMIT_REQUEST=8192
+ONBUILD ARG ODOO_LIMIT_TIME_REAL_CRON=-1
+ONBUILD ARG ODOO_DB_FILTER=""
+ONBUILD ARG ODOO_LIST_DB=1
+ONBUILD ARG ODOO_LOG_LEVEL=info
```

Comment:
Do you think we can reuse [image's LOG_LEVEL](https://github.com/Tecnativa/docker-odoo-base/pull/62/files#diff-3254677a7917c6c01f55212f86c57fbfR67)? Or do you prefer to keep them separate?

Reply:
IMO it would be best to keep them separate. The image log to me is a build/boot thing, vs the Odoo log being runtime. While they may be coordinated, they're for different people to use (the builder vs the maintainer)